bower.json

```diff
@@ -39,6 +39,7 @@
     "c3": "~0.4.11",
     "colorbrewer": "~1.0.0",
     "d3": "~3.5.17",
+    "d3v4": "https://d3js.org/d3.v4.min.js",
```

Comment:
I keep forgetting that this graph is dependent on d3v4 - how hard would it be to use d3v3 for the 1.20.x branch and then upgrade it to d3v4 for the 2.0.x?

Reply:
I'm not sure how hard it would be without diving into it.